### ISSUE TITLE
RainLoop affected by Mailsploit

### ISSUE THREAD

**Issue by BuZZ-dEE** [RainLoop/rainloop-webmail#1591]

**Comment by ervee** [comment]
Someone reported it there but it is not confirmed. Did you confirm it? Has anyone disclosed the issue to @RainLoop? Or are you just thinking out loud?

**Comment by xf-** [comment]
@ervee tested it with test utility from mailsploit.com and @RainLoop is also listed in Google Docs.

**Comment by BuZZ-dEE** [comment]
Yes, I can confirm it. I also tested it with test utility from mailsploit.com.

**Comment by ervee** [comment]
Okay. So far as I can tell this is a GUI sort-of-bug. The "From:" header used to display the e-mail source has always been easy to manipulate. This "bug" just adds the capability to circumvent SPF/DMARC if you have it configured in your MTA in the first place.

I don't know if there is anything @RainLoop can do about this but he can try.
I actually agree with Mozilla in their statement that this is to be fixed by the MTA. Not by the MUA.

**Comment by msberends** [comment]
Check the link; not affected anymore. Issue can be closed.
